com.unity.multiplayer.mlapi/Runtime/Core/NetworkedObject.cs

```diff
@@ -38,6 +38,10 @@ internal void ValidateHash()
             PrefabHash = PrefabHashGenerator.GetStableHash64();
         }
 
+        /// <summary>
+        /// Gets the NetworkingManager that owns this NetworkedObject instance
+        /// </summary>
+        public NetworkingManager NetworkManager => NetworkingManager.Singleton;
```

Comment:
this will (potentially) be `NetworkManager { get; internal set; }` in the future when we refactor `NetworkManager` and get rid of singleton pattern